File: scratch/analyze_settings.js
const fs = require('fs');
const path = require('path');

const filepath = path.join(__dirname, '..', 'views', 'settings.ejs');
const content = fs.readFileSync(filepath, 'utf8');
const lines = content.split('\n');

console.log(`Total lines: ${lines.length}`);
lines.forEach((line, idx) => {
  if (line.includes('id="panel-') || line.includes("id='panel-") || line.includes('settings-panel')) {
    console.log(`Line ${idx+1}: ${line.trim()}`);
  }
});

File: scratch/analyze_settings.py
import re

filepath = r"C:\e drive backup\Pictures\antiygravity\views\settings.ejs"
with open(filepath, 'r', encoding='utf-8') as f:
    lines = f.readlines()

print(f"Total lines: {len(lines)}")
for idx, line in enumerate(lines):
    if 'id="panel-' in line or "id='panel-" in line or 'settings-panel' in line:
        print(f"Line {idx+1}: {line.strip()}")
